react/client/src/styles/utilityStyles.ts

```diff
@@ -16,6 +16,7 @@ const useUtilityStyles = makeStyles((theme) =>
       maxWidth: '850px',
       marginLeft: 'auto',
       marginRight: 'auto',
+      marginTop: '49px',
```

Comment:
This is such an arbitrary value (though I suspect that's what the design has) but I'm good with it. We would want to use more common numbers that the entire app follows but this is so low priority that we can always come back to it.

Reply:
Yep you are right, I followed what the design has right now.
Yep that sounds good about the common numbers. Thanks for reviewing!